### DISCUSSION TITLE
Error while removing associations from Node 25: All node IDs must be between 1 and 232! (ZW0322)

### DISCUSSION THREAD

**Discussion by HubertusH** [zwave-js/zwave-js#3605] [Q&A] [answered]
i have a node with associations to node 243 and 255
![image](https://user-images.githubusercontent.com/16612146/139119237-861daace-f7f2-4733-8dcd-54ab6c2f5218.png)
if i delete them with the api call removeAssociation i get the error:
`2021-10-27 19:33:23.860 WARN ZWAVE: Error while removing associations from Node 25: All node IDs must be between 1 and 232! (ZW0322)
`
now the question is - how to remove this associations?

Thank you for your support
br
Hubertus

**Comment by AlCalzone** [comment, marked as answer]
-> #3606
